url/src/host.rs

```diff
@@ -90,29 +91,26 @@ impl Host<String> {
         }
 
         let is_invalid_domain_char = |c| {
-            matches!(
-                c,
-                | '\0'..='\u{001F}'
-                    | ' '
-                    | '#'
-                    | '%'
-                    | '/'
-                    | ':'
-                    | '<'
-                    | '>'
-                    | '?'
-                    | '@'
-                    | '['
-                    | '\\'
-                    | ']'
-                    | '^'
-                    | '\u{007F}'
-            )
+            matches!(c, |'\0'..='\u{001F}'| ' '
```

Comment:
This is an unrelated `cargo fmt` change that slipped through a previous PR. Whoops!